### PR TITLE
Add ETHx as collateral to WETH market on Mainnet

### PR DESCRIPTION
**Pull request by MishaShWoof**

### CODE REVIEW

File: deployments/mainnet/weth/migrations/1722603828_add_ethx_as_collaterals.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+export default migration('1722603828_add_ethx_as_collaterals', {
+  async prepare(deploymentManager: DeploymentManager) {
+    const _ETHxScalingPriceFeed = await deploymentManager.deploy(
+      'ETHx:priceFeed',
+      'pricefeeds/ScalingPriceFeed.sol',
+      [
+        ETHX_PRICE_FEED_ADDRESS,  // ETHx / ETH price feed
+        8                         // decimals
+      ]
```
Comment (Collapsence) [inline deployments/mainnet/weth/migrations/1722603828_add_ethx_as_collaterals.ts:17]: Add true

Comment (vitalii-woof-software) [inline deployments/mainnet/weth/migrations/1722603828_add_ethx_as_collaterals.ts:17]: Add true

File: deployments/mainnet/weth/migrations/1722603828_add_ethx_as_collaterals.ts [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
... = '# Add ETHx as collaterals into cWETHv3 on Mainnet\n\n## Proposal summary\n\nCompound Growth Program [AlphaGrowth] proposes to add ETHx into cWETHv3 on Ethereum network. This proposal takes the governance steps recommended and necessary to update a Compound III WETH market on Ethereum. Simulations have confirmed the market’s readiness, as much as possible, using the [Comet scenario suite](https://github.com/compound-finance/comet/tree/main/scenario). The new parameters include setting the risk parameters based on the [recommendations from Gauntlet](https://www.comp.xyz/t/listing-ethx-on-compound/4730/21).\n\nFurther detailed information can be found on the corresponding [proposal pull request](https://github.com/compound-finance/comet/pull/901) and [forum discussion](https://www.comp.xyz/t/listing-ethx-on-compound/4730/21).\n\n\n## Price feed\n\nExchange rate price feed of ETHx/ETH was provided by Chainlink team. The address of pricefeed that is used is 0xdd487947c579af433AeeF038Bf1573FdBB68d2d3\n\n\n## Proposal Actions\n\nThe first proposal action adds ETHx asset as collateral with corresponding configurations.\n\nThe second action deploys and upgrades Comet to a new version.';
```
Comment (vitalii-woof-software) [inline deployments/mainnet/weth/migrations/1722603828_add_ethx_as_collaterals.ts:71]: Change description

**Review by torreyatcitty — APPROVED** [review]
LGTM

**Pull request merged**